### PR TITLE
 NVIDIA MLPerf v0.5.0 - Mask R-CNN pretrained RN50 backbone conversion

### PR DESCRIPTION
**Pull request by cliffwoolley**
Mask R-CNN conversion script to translate the reference Caffe2 pretrained RN50 backbone to PyTorch format.

Note that `maskrcnn_benchmark/utils/c2_model_loading.py` in the original NVIDIA submission already contained what's essentially this code (from https://github.com/facebookresearch/maskrcnn-benchmark/blob/optimized/maskrcnn_benchmark/utils/c2_model_loading.py).  This PR merely brings in the minor tweak to that script that we had used to convert the model offline as opposed to online.

### CODE REVIEW

**Comment by nvpaulius** [comment]
merging given the approval above
